### PR TITLE
PMK-5674 Adding support for Rocky Linux 9

### PR DESCRIPTION
**Pull request by manasabsv26**
## ISSUE(S):
[PMK-5674](https://platform9.atlassian.net/browse/PMK-5674)

## SUMMARY
Added support for Rocky Linux 9.1. 

## ISSUE TYPE
- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

#### Manual

Was able to onboard Rocky 9.1 node to the DU and create 1.25 cluster with the node, all addons are installed and are healthy.
Didn't see any errors in hostagent/nodelet logs. Was able to delete the cluster. 

image

`Addons`:

image

[PMK-5674]: https://platform9.atlassian.net/browse/PMK-5674?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pkg/platform/centos/centos.go [hunk 1]
```diff
@@ -295,7 +302,7 @@ func (c *CentOS) Version() (string, error) {
 	//using cut command required field (7.6.1810) is selected.
 	var cmd string
 
-	_, err := c.exec.RunWithStdout("bash", "-c", "grep -i 'centos' /etc/*release")
+	_, err := c.exec.RunWithStdout("bash", "-c", "grep -i 'CentOS Linux' /etc/*release")
```
Comment (psarwate) [inline pkg/platform/centos/centos.go:305]: Why is this change needed?

Comment (manasabsv26) [reply pkg/platform/centos/centos.go:305]: Rocky's /etc/os-release file has this line:
ID_LIKE="rhel centos fedora", so to differenciate between centos and rocky machines, added this line.

**Review by psarwate — APPROVED** [review]
/lgtm
